app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module SessionsHelper
```

Comment:
Helper files are to be used with Views. Any logic having to do with controllers should be placed either in Application Controller or in Concerns. [REQUIRED]